src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/ExtendedQueryTagString.sql

```diff
@@ -21,19 +28,21 @@ CREATE UNIQUE CLUSTERED INDEX IXC_ExtendedQueryTagString ON dbo.ExtendedQueryTag
     TagKey,
     TagValue,
     PartitionKey,
-    StudyKey,
-    SeriesKey,
-    InstanceKey
+    ResourceType,
```

Comment:
Should'nt resourceType be the first column? It is kind least specific value

Reply:
Make sense, will move to the top